app/models/concerns/geoblacklight/solr_document.rb

```diff
@@ -45,7 +45,11 @@ def oembed
     end
 
     def same_institution?
-      fetch(Settings.FIELDS.PROVENANCE).casecmp(Settings.INSTITUTION.downcase).zero?
+      if self.has_field?(Settings.FIELDS.PROVENANCE)
+        fetch(Settings.FIELDS.PROVENANCE).casecmp(Settings.INSTITUTION.downcase).zero?
```

Comment:
I think `fetch` allows you to provide a default value as an argument which could help remove the double conditional

Reply:
Okay, this _could_ be rewritten as such:

```ruby
    def same_institution?
      fetch(
        Settings.FIELDS.PROVENANCE,
        self.has_field?(Settings.FIELDS.PROVENANCE)
      )
      .to_s
      .casecmp(
        Settings.INSTITUTION.downcase
      ).zero?
    end
```

But, I honestly prefer the if/else statement because it is explicit. Chaining the .to_s to fetch (to support fetch returning false) to pass along to .casecmp feels a bit mumbo jumbo.

@mejackreed let me know which way you prefer and we'll go that route.

Reply:
What about just this?

```diff
def same_institution?
-  fetch(Settings.FIELDS.PROVENANCE).casecmp(Settings.INSTITUTION.downcase).zero?
+  fetch(Settings.FIELDS.PROVENANCE, '').casecmp(Settings.INSTITUTION.downcase).zero?
end
```

Reply:
That's a winner. 😄